horizons/world/player.py

```diff
@@ -38,7 +38,16 @@
 
 
 class Player(ComponentHolder, WorldObject):
-	"""Class representing a player"""
+	"""Class representing a player
+
+	@type session: horizons.session.Session
+	@param session: Session instance
+	@param worldid: player's worldid
+	@param name: user-chosen name
+	@param color: color of player (as Color)
+	@param clientid: id of client
+	@param inventory: {res: value} that are put in the players inventory
+	"""
```

Comment:
All this tells is, that sometimes `__init__` is not documented. It doesn't say anything about parameters documented directly at the beginning of a class definition.